tests/nested/manual/uc20-fde-hooks-v1/task.yaml

```diff
@@ -0,0 +1,42 @@
+summary: Check that the fde-setup hooks work
```

Comment:
The PR is about hooks v2, but the test is about hooks v1, is there a v2 test coming separately?

Reply:
This should also be clarified in the followup, the "uc20-fde-hooks" test is the v2 test and v2 is really what everyone should be using from this PR on. I will update the test description to make this clearer